packages/deposit_supabase/lib/src/supabase_deposit_adapter.dart

```diff
@@ -90,6 +99,15 @@ class SupabaseDepositAdapter extends DepositAdapter<int> {
     }
   }
 
+  @override
+  Future<void> removeAll(
+    String table,
+    String primaryColumn,
+    List<Map<String, dynamic>> data,
+  ) async {
```

Comment:
```suggestion
  ) {
```

Reply:
This one has to stay async cause the Future.wait is not returned, but you do want to wait until it is finished